### DISCUSSION TITLE
More apps in home

### DISCUSSION THREAD

**Discussion by somedeveloper00** [tanujnotes/Olauncher#362] [Ideas]
Having to search for them is tedious when it happens more than a few times every day. It would be really good if we could add even more apps in the home so that we could just tap on them to start them, preferably not taking away the alignment feature (so we can still align them to i.e. bottom-right for simple thumb click) 

thanks

**Comment by tanujnotes** [comment]
No plan to add more apps on the home screen. What you can do instead is rename the apps in a way that they appear on the top of the app drawer.
